langchain/src/retrievers/multi_retriever.ts

```diff
@@ -0,0 +1,95 @@
+import { BaseRetriever, BaseRetrieverInput } from "@langchain/core/retrievers";
+import { Document } from "@langchain/core/documents";
+import { CallbackManagerForRetrieverRun } from "@langchain/core/callbacks/manager";
+import { BaseDocumentCompressor } from "@langchain/core/retrievers/document_compressors";
+
+export interface MultiRetrieverInput extends BaseRetrieverInput {
+  retrivers: BaseRetriever[];
+  documentCompressor?: BaseDocumentCompressor | undefined;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  documentCompressorFilteringFn?: (docs: Document<Record<string, any>>[]) => Document<Record<string, any>>[];
+}
+
+export class MultiRetriever extends BaseRetriever {
```

Comment:
Can we rename to `EnsembleRetriever` to match Python?

https://python.langchain.com/v0.2/docs/how_to/ensemble_retriever/